index.js

```diff
@@ -350,6 +350,7 @@ function noop () {}
 
 function toBuffer (str) {
   if (typeof str === 'string') return Buffer.from(str, 'hex')
+  if (ArrayBuffer.isView(str)) return Buffer.from(str.buffer, str.byteOffset, str.byteLength)
```

Comment:
I'm going to move the line below this one above this check. Since I think `ArrayBuffer.isView(someBuffer)` will be true